src/providers/jira.rs

```diff
@@ -15,7 +15,13 @@ pub struct JiraConfiguration {
     pub prefix: Option<String>,
 }
 
-pub async fn get_ticket(id: String, configuration: &JiraConfiguration) -> Result<Value, Error> {
+pub async fn get_ticket(
+    id: String,
+    configuration: &JiraConfiguration,
+) -> Result<Option<Value>, Error> {
+    if id == "skip" { // sory for this, lest think in a better strategy to deal with this case
+        return Ok(None);
+    }
     let url = format!(
```

Comment:
What do you think about make it optional into the [`providers/mod.rs`](https://github.com/compiled-thoughts/git-ai/blob/0304680e18cf690afcbc6e7b818363e6b9d250d8/src/providers/mod.rs)?